File: app/src/at-trinhnguyen/java/asiantech/internship/winter/javacore/oop/Circle.java
package asiantech.internship.winter.javacore.oop;

public class Circle extends Shape {
    private double radius;

    public Circle(double radius) {
        this.radius = radius;
    }

    @Override
    public double area() {
        return 3.14 * radius * radius;
    }

    @Override
    public double circumference() {
        return 3.14 * radius * 2;
    }


    @Override
    public void print() {
        System.out.print("Circle ");
    }
}

File: app/src/at-trinhnguyen/java/asiantech/internship/winter/javacore/oop/Circle.kt
package asiantech.internship.winter.javacore.oop

class Circle(private val radius: Double) : Shape() {

    public override fun area(): Double {
        return Math.PI * radius * radius
    }

    public override fun circumference(): Double {
        return Math.PI * radius * 2.0
    }


    public override fun print() {
        print("Circle ")
    }
}
